Partner Manager - Reorg of architecture and trading partner topics (#139)
mulesoft/docs-partner-manager: create-partner-lp.adoc

> = Work with Partners

Reviewer: In this case, I think we might be able to use a more specific title than just "work with," because this is actually about  creating a new partner and an information exchange workflow with that new partner.
And I LOVE that you added "partner," and limited directionality to "send" and "receive" from the partner perspective rather than "inbound" and "outbound," which was super confusing to me.

Reviewer: Ah! nevermind...I see that this is all replaced below.

> In order to exchange data with a company, you must first set it up as a partner. Partner Manager gives you the ability to create and manage partner details such as 

Reviewer: ```suggestion
To exchange data with a company, use Partner Manager to first set up that company as a _partner_ and then to 
 manage partner details such as 
```

Reviewer: ditto: replaced
